pysindy/_core.py

```diff
@@ -177,31 +227,25 @@ class SINDy(_BaseSINDy):
 
     Parameters
     ----------
-    optimizer
+    optimizer : optimizer object, optional
         Optimization method used to fit the SINDy model. This must be a class
         extending :class:`pysindy.optimizers.BaseOptimizer`.
-        The default is :class:`pysindy.optimizers.STLSQ`.
+        The default is :class:`STLSQ`.
 
-    feature_library
+    feature_library : feature library object, optional
         Feature library object used to specify candidate right-hand side features.
         This must be a class extending
         :class:`pysindy.feature_library.base.BaseFeatureLibrary`.
-        The default option is :class:`pysindy.feature_library.PolynomialLibrary`.
+        The default option is :class:`PolynomialLibrary`.
 
-    differentiation_method
+    differentiation_method : differentiation object, optional
         Method for differentiating the data. This must be a class extending
-        :class:`pysindy.differentiation.base.BaseDifferentiation` class.
+        :class:`pysindy.differentiation_methods.base.BaseDifferentiation` class.
```

Comment:
Don't add the `: differentiation object, optional`.  sphinx picks this up from the type annotation in `__init__`.

No need to clean up everywhere, just don't add it.

Also, revert the `differentiation` -> `differentiation_methods`